libs/tree-sitter-titan/grammar.js

```diff
@@ -1,7 +1,42 @@
 module.exports = grammar({
-  name: 'titan',
-
+  name: "titan",
   rules: {
-    source_file: $ => 'hello'
-  }
-});
\ No newline at end of file
+    variable_definition: ($) =>
+      seq(
+        $._variable_start,
+        field("pattern", $.alpha_identifier),
+        optional(seq(":", field("type", $._type))),
+        "=",
+        field("value", $._non_null_literal)
```

Comment:
Should we end statements in semicolon?

Reply:
I'd like to make semicolon optional like how Scala does.